src/components/App.jsx

```diff
@@ -20,7 +23,7 @@ function App() {
       <CreateArea onClick={addNote} />
       {notes.map((note, index) => (
         <Note
-          key={uniqid()}
```

Comment:
As we stop using the `uniqid` it is a good practice to clean imports and remove this package from the project